include/pybind11/pytypes.h

```diff
@@ -514,7 +514,6 @@ class accessor : public object_api<accessor<Policy>> {
         return cache;
     }
 
-private:
```

Comment:
Accident?
If not, could you please add a comment why the members below cannot be private anymore?

Reply:
This was a duplicated private, there's one just 10 lines above or so. However, it was removed to try to fix sphinx+breathe errors, which I finally fixed by forcing and old sphinx and old breathe.